### PR TITLE
Uar 167 skeletal update submitted page

### PR DESCRIPTION
**Pull request by garywco**
### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Review by corybudd-ch — COMMENTED** [review]
Have to remove package-lock.json changes from your branch and use main's by running the below:

git rm --cached package-lock.json 
git checkout main -- package-lock.json 

then push.

File: routes.yaml [hunk 1]
```diff
@@ -4,4 +4,4 @@ group: web
 weight: 100
 routes:
   1: ^/register-an-overseas-entity/*
-  2: ^/update-an-overseas-entity/*
+  2: ^/update-an-overseas-entity/*
```
Comment (corybudd-ch) [inline routes.yaml:7]: can't tell what the difference is here! But checkout the routes.yaml from main and push to your branch to get rid of this line change.

File: src/controllers/update/update.confirmation.controller.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.UPDATE_CONFIRMATION_URL}`);
+    return res.render(config.UPDATE_CONFIRMATION_PAGE, {
+      backLinkUrl: config.UPDATE_LANDING_PAGE_URL,
```
Comment (corybudd-ch) [inline src/controllers/update/update.confirmation.controller.ts:10]: UPDATE_LANDING_PAGE_URL is incorrect for the back link as that's the first page in the journey, but not sure what your back link should be. Probably something to check with David or Matthew from product.

File: src/utils/navigation.ts [hunk 3]
```diff
@@ -25,6 +25,11 @@ export const NAVIGATION: Navigation = {
     previousPage: () => config.UPDATE_LANDING_PAGE_URL,
     nextPage: [config.CONFIRM_OVERSEAS_ENTITY_DETAILS_PAGE]
   },
+  [config.UPDATE_CONFIRMATION_URL]: {
+    currentPage: config.UPDATE_CONFIRMATION_PAGE,
+    previousPage: () => config.OVERSEAS_ENTITY_QUERY_URL,
+    nextPage: [config.UPDATE_LANDING_PAGE_URL]
```
Comment (corybudd-ch) [inline src/utils/navigation.ts:31]: previousPage and nextPage seem incorrect. I think nextPage can just be [] if there is nothing after your page. previousPage not sure what should be

File: test/setup.ts [hunk 4]
```diff
@@ -13,6 +13,7 @@ export default () => {
   process.env.PAYMENT_FEE = "100";
   process.env.LANDING_PAGE_URL = "/register-an-overseas-entity/sold-land-filter?start=0";
   process.env.UPDATE_LANDING_PAGE_URL = "/update-an-overseas-entity/overseas-entity-query";
+  process.env.UPDATE_CONFIRMATION_LANDING_PAGE_URL = "/update-an-overseas-entity-confirmation";
```
Comment (corybudd-ch) [inline test/setup.ts:16]: this isn't required I don't think, shouldn't be any new env variables being added UPDATE_CONFIRMATION_LANDING_PAGE_URL

File: routes.yaml (hunk 1 above)
Comment (stephanie-ch) [reply routes.yaml]: a newline is also needed at the end of this file

Comment (garywco) [reply routes.yaml]: Restored package-lock.json and routes.yaml

**Review by stephanie-ch — CHANGES_REQUESTED** [review]

File: test/setup.ts (hunk 4 above)
Comment (garywco) [reply test/setup.ts:16]: Done

**Comment by AndreKramer7654** [comment]
This PR will be handled as part of https://github.com/companieshouse/overseas-entities-web/pull/589. Please review and comment there.

**Comment by AndreKramer7654** [comment]
No longer needed as page merged in other PR.

**Pull request closed without merging**